cognite_toolkit/_cdf_tk/commands/deploy.py

```diff
@@ -155,12 +155,6 @@ def execute(
                 has_dropped_data=drop_data,
                 verbose=ctx.obj.verbose,
             )
-            if ToolGlobals.failed:
-                if results and results.has_counts:
-                    print(results.counts_table())
-                if results and results.has_uploads:
-                    print(results.uploads_table())
-                raise ToolkitDeployResourceError(f"Failure to load/deploy {loader_instance.display_name} as expected.")
```

Comment:
This is no longer necessary as we know raise an appropriate exception in the place the error occurred, instead of the generic one here.